netbox/netbox/tables/columns.py

```diff
@@ -510,25 +511,32 @@ def __init__(self, customlink, *args, **kwargs):
 
         super().__init__(*args, **kwargs)
 
-    def render(self, record):
-        try:
-            rendered = self.customlink.render({
-                'object': record,
-                'obj': record,  # TODO: Remove in NetBox v3.5
+    def _render_customlink(self, record, table):
+        context = {
+            'object': record,
+            'obj': record,  # TODO: Remove in NetBox v3.5
```

Comment:
Shouldn't this be removed now?

Reply:
It should have been removed in v3.5, yes, but we didn't, so I've retained it in my PR. It _has_ been removed in `feature` already for the upcoming v3.6 release.